code/kassaSysteem/database/seeders/UserSeeder.php

```diff
@@ -14,27 +14,28 @@ class UserSeeder extends Seeder
      */
     public function run(): void
     {
-        DB::table('user_ids')->insert(
+        DB::table('users')->insert(
             [
                 [
                     'naam' => 'Jef',
-                    'password' => 'Jef',
+                    'wachtwoord' => Hash::make('Jef'),
                     'rol_id' => 2,
-                    'organisatie_id' => 2
-
+                    'organisatie_id' => 2,
+                    'wachtwoordWijzigen' => 0
```

Comment:
wachtwoordWijzigen is extra toegevoegd. Als deze true is zal de gebruiker verplicht zijn wachtwoord moeten veranderen.